kong-1.0.3-0.rockspec

```diff
@@ -236,6 +236,7 @@ build = {
     ["kong.plugins.response-ratelimiting.migrations.000_base_response_rate_limiting"] = "kong/plugins/response-ratelimiting/migrations/000_base_response_rate_limiting.lua",
     ["kong.plugins.response-ratelimiting.migrations.001_14_to_15"] = "kong/plugins/response-ratelimiting/migrations/001_14_to_15.lua",
     ["kong.plugins.response-ratelimiting.migrations.002_15_to_10"] = "kong/plugins/response-ratelimiting/migrations/002_15_to_10.lua",
+    ["kong.plugins.response-ratelimiting.migrations.003_index"] = "kong/plugins/response-ratelimiting/migrations/003_index.lua",
```

Comment:
We shouldn't shy away with shipping migrations in patch releases (optional ones, that is, exactly like this one). Nothing guarantees that a hotfix won't require a migration at some point (this has most likely already happened in the last 4 years, but my memory isn't good enough to recall it).

And since all deployments of Kong ideally _should_ automate `kong migrations list` and follow the follow dictated by its exit code, migrations should (theoretically...) be ran automatically. Not shipping migrations in minor and patch releases was only due to our lack of confidence in the old migrations framework. If our confidence is too low with the current framework, we should improve it and feel safe shipping migrations at all times (I think even with the couple of flaws we know about the new framework and will eventually fix, we already feel a lot safer with it compared to the old one).

Reply:
@mvanholsteijn in light on @thibaultcha's comments, could you rebase this to `master` and rename `03_index` to `03_10_to_112`? Thank you in advance!

Reply:
do you need `03_10_to_112` or `03_10_to_12`?

Reply:
`03_10_to_112` and rebased to `master`, as we want to get this out sooner, in Kong 1.1.2!

Reply:
done.